docs/source/reference/glossary.rst

```diff
@@ -181,3 +181,26 @@ tag
 TensorBay SDK has the ability to tag the specific :ref:`reference/glossary:commit` in a dataset's history
 as being important. Typically, people use this functionality to mark release points (v1.0, v2.0 and so on).
 
+
+branch
+======
+
+Similar to git, a branch is a lightweight pointer to one of the commits.
+
+Currently, TensorBay only supports a single branch, which is the default "main" branch.
+
+As :ref:`commits<reference/glossary:commit>` are submitted, the main branch will point to the lastest commit being made.
+Every time a commit is made, the main branch pointer moves forward automatically.
+
+
+revision
+========
+
+Similar to Git, a revision is a reference to a single :ref:`reference/glossary:commit`.
+And many methods in TensorBay SDK take revision as an argument.
+
+Currently, a revision can be in the following forms:
+
+1. The full commit ID, which is a uuid in 36-byte hexadecimal string.
```

Comment:
do we need to add "commit ID" to glossary?
@Hoteryoung

Reply:
The -> A
There can a brief introduction about the commit ID in the `commit` item of the glossary page.
Thus, here, we can write : `A full commit ID.`